### ISSUE TITLE
Error message for no more spies in supply

### ISSUE THREAD

**Issue by dotdotdot-tts** [rogeriocamorim/duneBugSuggestionTracker#28]
new error message suggestion:
- error message for no more spies in supply when trying to add a spy when 3 are already on the board.

**Comment by aaittouares** [comment]
since we got this kind of message for agents, it's coherent we have this for spies, but honestly zero priority on this since it's pretty obvious why clicking on a observation post did nothing .....
It will be added sometime in the future with no ETA.

**Comment by dotdotdot-tts** [comment]
yeah just mentioning for consistency, obviously very very very low priority haha
